lib/graph-test.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+ <html>
+
+ <head>
+
+     <title>Auditly</title>
+     <script src="../node_modules/force-graph/dist/force-graph.min.js"></script>
```

Comment:
Instead of adding from node_modules, copy this file from node_modules to the output file.  Generally, everything the user needs to see the output should be in the output folder.

Reply:
When this file is output, the path works as expected. We should have a dist folder in the output to hold any JS however, so we don't need to reference node modules, but have everything packaged where its needed.